src/molecules/Chip.tsx

```diff
@@ -1,34 +1,45 @@
 import React from 'react';
-import { StyleSheet, View, ViewStyle, Text } from 'react-native';
+import { FlexStyle, StyleProp, StyleSheet, View, ViewStyle, Text } from 'react-native';
 import { registeredTextStyle } from 'src/utils/textStyles';
 import { colors } from 'src/utils/colors';
 
 const styles = StyleSheet.create({
   Wrapper: {
-    alignSelf: 'flex-start', // the only way to make sure that the element won't grow
     paddingHorizontal: 4,
     borderRadius: 7,
     height: 14,
     backgroundColor: colors.tertiary, // default bgState maps to this
   } as ViewStyle,
-  Text: registeredTextStyle('micro', colors.ulisse, 'chip-text'),
 });
 
 export type ChipProps = {
   label: string;
   bgState: 'default' | 'success' | 'error';
+  bgColor?: 'ulisse' | 'brand';
```

Comment:
I added the prop bgColor to handle the style `ulisse` and `brand`. Thanks

https://github.com/urbi-mobility/urbi-react-native/commit/19edb04586a452a10778b9d3a20b43608c60d424